app/src/main/java/com/kks/nimblesurveyjetpackcompose/viewmodel/survey/SurveyDetailViewModel.kt

```diff
@@ -24,6 +24,7 @@ class SurveyDetailViewModel @Inject constructor(
     private val _surveyQuestions = MutableStateFlow(emptyList<SurveyQuestion>())
     private val _shouldShowLoading = MutableStateFlow(false)
     private val _error = MutableStateFlow<ErrorModel?>(null)
+    private val _shouldShowLottie = MutableStateFlow(false)
```

Comment:
I mean, we should make it abstract here as well. 😄 
```suggestion
    private val _shouldShowThanks = MutableStateFlow(false)
```